src/main/java/vendored/org/apache/dolphinscheduler/common/utils/process/ProcessImplForWin32.java

```diff
@@ -811,12 +879,19 @@ private static int getExitCodeProcess(WinNT.HANDLE handle) {
     }
 
     private static void terminateProcess(WinNT.HANDLE handle) {
-        Kernel32.INSTANCE.TerminateProcess(handle, 1);
+        if (!Kernel32.INSTANCE.TerminateProcess(handle, EXIT_CODE_TERMINATED)) {
+            logger.warn("Terminate process failed {}", Kernel32Util.getLastErrorMessage());
+        }
     }
 
     private static boolean isProcessAlive(WinNT.HANDLE handle) {
         IntByReference exitStatus = new IntByReference();
-        Kernel32.INSTANCE.GetExitCodeProcess(handle, exitStatus);
+        if (!Kernel32.INSTANCE.GetExitCodeProcess(handle, exitStatus)) {
+            int errorCode = Kernel32.INSTANCE.GetLastError();
+            logger.info("GetExitCodeProcess failed {} {}", handle,
+                    Kernel32Util.formatMessageFromLastErrorCode(errorCode));
```

Comment:
Don't log the handle, I only added that because of the weird stuff we saw before.